hydpy/exe/servertools.py

```diff
@@ -823,9 +823,11 @@ class HydPyServer(http.server.BaseHTTPRequestHandler):
 
     >>> for element in ("land_lahn_1", "land_lahn_2"):
     ...     sequences = f"HydPyServer.state.hp.elements.{element}.model.sequences"
+    ...     submodel_sequences = (f"HydPyServer.state.hp.elements.{element}.model."
+    ...                           f"rconcmodel.sequences")
     ...     test("evaluate",
     ...          data=(f"sm = {sequences}.states.sm \\n"
-    ...                f"quh = {sequences}.logs.quh"))  # doctest: +ELLIPSIS
+    ...                f"quh = {submodel_sequences}.logs.quh"))  # doctest: +ELLIPSIS
```

Comment:
Maybe a little more readable:

```python
    >>> for element in ("land_lahn_1", "land_lahn_2"):
    ...     path_element = f"HydPyServer.state.hp.elements.{element}"
    ...     path_sequences_model = f"{path_element}.model.sequences"
    ...     path_sequences_submodel = f"{path_element}.model.rconcmodel.sequences")
    ...     test("evaluate",  # doctest: +ELLIPSIS
    ...          data=(f"sm = {path_sequences_model}.states.sm \\n"
    ...                f"quh = {path_sequences_submodel}.logs.quh"))
```

Reply:
Note that I moved the "# doctest: +ELLIPSIS" statement to another line, so that it does not violate the maximum line length.  (Otherwise, Pylint would emit a `line-too-long` error.)

Reply:
Is corrected.